packages/ui/src/styles/globals.css

```
@@ -258,6 +260,109 @@
+    in oklch,
+    var(--color-foreground-primary) 4%,
+    transparent
+  );
+  --surface-frame-elevation: 0 4px 12px -4px
+    color-mix(in oklch, var(--color-black) 5%, transparent);
+  box-shadow:
+    inset 0 var(--surface-frame-bevel-y) 0 var(--surface-frame-bevel-color),
+    0 0 0 var(--surface-frame-ring-spread) var(--surface-frame-ring-color),
+    0 0 0 var(--surface-frame-halo-gap) var(--surface-frame-halo-spacer),
+    0 0 0 var(--surface-frame-halo-spread) var(--surface-frame-halo-color),
+    var(--surface-frame-elevation);
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)